static/js/components/Navbar.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+import React from 'react';
+import UserMenu from '../containers/UserMenu';
+import { HeaderTabs, Header, HeaderRow, Tab } from 'react-mdl';
+
+class Navbar extends React.Component {
+  props: {
+    empty:      boolean,
+    children?:  React$Element[],
+    pathname:   string,
+    changeUrl:  (i: number) => void,
+  };
+
+  makeTabs: Function = (): React$Element[] => (
+    this.tabs.map((tab, i) => <Tab key={i}>{tab.label}</Tab>)
+  );
+
+  tabs: Object[] = [
+    { label: 'Dashboard', path: '/dashboard', regex: /dashboard/ },
+    { label: 'Users', path: '/users', regex: /users/ }
+  ];
+
+  activeTab: Function = (path: string): number => (
+    this.tabs.findIndex(tab => tab.regex.test(path))
+  );
+
+  userMenu: Function = (): void|React$Element => {
+    const { empty } = this.props;
```

Comment:
this is basically just ported over from the old `Header` component.

basically, if we set `empty` it means that the navbar shouldn't show the user menu. we use it on the terms of service page.